gallery/how_to/work_with_microtvm/install_cmsis.rst

```diff
@@ -1,35 +0,0 @@
-..  Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Please don't remove this file as we will need it in future tutorials

Reply:
I re-added the file and edited it adding the commands to download CMSIS NN from the new GitHub location.